submissions/solaryasha/html-css-popup/css/styles.css

```diff
@@ -0,0 +1,193 @@
+body {
+    font-size: 15px;
+    font-family: 'Segoe UI', 'Roboto', arial, sans-serif;
+    overflow: hidden;
+}
+
+* {
+    box-sizing: border-box;
+}
+.navigation {
+    display: block;
+    height: auto;
+}
+
+.nav_menu {
+    margin-right: 20px;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+}
+.menu_card {
+    display: flex;
+    align-items: center;
+    list-style-type: none;
+    margin-left: 20px;
+}
+.menu_card a {
+    text-decoration: none;
+    color: black;
+}
+.menu_card img {
+    width: 32px;
+    height: 32px;
+}
+.avatar img {
+    width: 32px;
+    height: 32px;
+}
+.iconsBox {
+    display: none;
+    position: absolute;
+    width: 300px;
+    height: 480px;
+    top: 70px;
+    right: 50px;
+    background: #fff;
+    border: 1px solid #ccc;
+    border-color: rgba(0,0,0,.2);
+    box-shadow: 0 2px 10px rgba(0,0,0,.2);
+    overflow-y: auto;
+    overflow-x: hidden;
+}
+
+.visibleIcons,
+.hiddenIcons {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    padding-left: 0px;
+    list-style-type: none;
+}
+.popupItem,
+.hidden {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 80px;
+    height: 80px;
+    margin-top: 20px;
+    padding: 5px 5px 0 5px;
+}
+.popupItem a {
+    text-decoration: none;
+    margin-top: 5px;
+    margin-left: 5px;
+    text-align: center;
+}
+.popupItem img,
+.hiddenIcons img {
+    display: flex;
+    
+    justify-content: center;
+    width: 40px;
+    height: 40px;
+    margin-left: auto;
+    margin-right: auto;
+}
+.popupItem figure {
+    margin: 10px;
+}
+.menu-icon {
+    background-image: url(../img/apps.jpg);
+    background-size: cover;
+    height: 24px;
+    width: 24px;
+    cursor: pointer;
+}
+.bell img{
+    width: 20px;
+    height: 20px;
+}
+.menu-btn,
+.moreApps {
+    position: absolute;
```

Comment:
Watch video how to hide checkboxes in right way. [Video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s)

Reply:
Fix it with the CSS property appearance.